srcpkgs/la-capitaine-icon-theme/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'la-capitaine-icon-theme'
+pkgname=la-capitaine-icon-theme
+version=0.6.1
+revision=1
+archs=noarch
+build_style=fetch
+short_desc="Icon pack designed to integrate with most desktop environments"
+maintainer="linarcx <linarcx@gmail.com>"
+license="GPL-3"
+homepage="https://github.com/keeferrourke/${pkgname}"
```

Comment:
Replace all ${pkgname} with its value

Reply:
Why?

Reply:
- Address can not be just copied out of template and pasted into browser.
- This is effectively constant so employing variable substitution is a waste.